locales/en/product-view.json

```diff
@@ -0,0 +1,3 @@
+{
+  "productDetail": "Product Details"
```

Comment:
I was wondering if we can use the same word "detail" without 's' ☺️ Btw, in terms of grammar, both are correct.